docs/core_systems/service_registry.md

```diff
@@ -173,7 +173,105 @@ Auto initialization of database tables. Value must be always 'none'.
 
 ### Custom parameters
 
-TODO
+:fontawesome-solid-wrench: **authentication.policy**
+
+Way of authenticating the service requester systems. Can be `declared`, `certificate` or `outsourced`.
+Learn more about the [authentication policies](../api/authentication_policy.md).
+
+:fontawesome-solid-wrench: **authenticator.credentials**
+
+The credentials what this system will use for performing the login operation when the authentication policy is `outsourced`.
+
+```
+authenticator.credentials={\
+	'<credential-name>': '<credential-value>' \
+}
+```
+
+:fontawesome-solid-wrench: **authenticator.secret.keys**
+
+Secret key for the authenticator servers when authentication policy is `outsourced`. The authenticator servers are able to register their authentication related services by providing their associated secret keys.
```

Comment:
> providing their associated secret keys.

This is not true. Actually, it provides its hashed name. The hashing algorithm uses the secret key.